### ISSUE TITLE
[REQUEST] multiple dynamic loss scalar on different loss values

### ISSUE THREAD

**Issue by amsword** [microsoft/DeepSpeed#1384]
**Is your feature request related to a problem? Please describe.**
We have different losses in each iteration, and we'd like to have different dynamic loss scalar to stable the iteration. However, the current deepspeed only supports one scalar if I understand correctly.

**Describe the solution you'd like**
I'm not sure how deepspeed can implement this feature, but a simple apex implementation can be as follows by specifying different loss id.
```python
with amp.scale_loss(losses, self.optimizer, loss_id=loss_id) as scaled_loss:
   scaled_loss.backward()
```

**Describe alternatives you've considered**
Currently, for stability, we used apex, which does not have the feature of ZERO.
